test/sensitivities/ambiguity_resolution.jl

```diff
@@ -0,0 +1,9 @@
+@testset "Ambiguity Resolution" begin
+    @testset "fill" begin
+        k = fill(Leaf(Tape(), 1.2), 3, 4)
```

Comment:
The ambiguity was coming up for the 1D case e.g.
```
k = fill(Leaf(Tape(), 1.2), 3)
```
Is it worth testing that explicitly too?

Reply:
I don't think it is.
Since it hits the same code.
and it is indeed fixed by this as the error in CI didn't occur